main/src/main/res/values/strings.xml

```diff
@@ -7,79 +7,79 @@
+    <string name="cant_read_folder">Could not read directory</string>
     <string name="select">Select</string>
     <string name="cancel">Cancel</string>
     <string name="no_data">No Data</string>
     <string name="useLZO">LZO Compression</string>
     <string name="client_no_certificate">No Certificate</string>
     <string name="client_certificate_title">Client Certificate</string>
     <string name="client_key_title">Client Certificate Key</string>
-    <string name="client_pkcs12_title">PKCS12 File</string>
+    <string name="client_pkcs12_title">PKCS 12 File</string>
     <string name="ca_title">CA Certificate</string>
-    <string name="no_certificate">You must select a certificate</string>
-    <string name="copyright_guicode">Source code and issue tracker available at https://github.com/schwabe/ics-openvpn/</string>
-    <string name="copyright_others">This program uses the following components; see the source code for full details on the licenses</string>
+    <string name="no_certificate">Select a certificate</string>
+    <string name="copyright_guicode">Source code and issue tracker at https://github.com/schwabe/ics-openvpn/</string>
+    <string name="copyright_others">This program uses the following components; the source code has full details on licenses</string>
     <string name="about">About</string>
     <string name="vpn_list_title">Profiles</string>
     <string name="vpn_type">Type</string>
-    <string name="pkcs12pwquery">PKCS12 Password</string>
+    <string name="pkcs12pwquery">PKCS 12 Password</string>
     <string name="file_select">Select…</string>
-    <string name="file_nothing_selected">You must select a file</string>
+    <string name="file_nothing_selected">Select a file</string>
     <string name="useTLSAuth">Use TLS Authentication</string>
     <string name="tls_direction">TLS Direction</string>
-    <string name="ipv6_dialog_tile">Enter IPv6 Address/Netmask in CIDR Format (e.g. 2000:dd::23/64)</string>
-    <string name="ipv4_dialog_title">Enter IPv4 Address/Netmask in CIDR Format (e.g. 1.2.3.4/24)</string>
+    <string name="ipv6_dialog_tile">Enter IPv6 address/netmask in CIDR format (e.g. 2000:dd::23/64)</string>
+    <string name="ipv4_dialog_title">Enter IPv4 address/netmask in CIDR format (e.g. 1.2.3.4/24)</string>
     <string name="ipv4_address">IPv4 Address</string>
     <string name="ipv6_address">IPv6 Address</string>
-    <string name="custom_option_warning">Enter custom OpenVPN options. Use with caution. Also note that many of the tun related OpenVPN settings cannot be supported by design of the VPNSettings. If you think an important option is missing contact the author</string>
+    <string name="custom_option_warning">Enter custom OpenVPN options. Use with caution. Also note that many of the TUN related OpenVPN settings cannot be supported by design of the VPNSettings. Contact the author if you think an important option is missing.</string>
```

Comment:
no tun is almost always written small.

Reply:
That seems to be in error though
https://www.kernel.org/doc/Documentation/networking/tuntap.txt
http://vtun.sourceforge.net/tun/faq.html
https://www.sobyte.net/post/2022-04/tun-mode
https://www.wintun.net/